tests/integration/utils.go

```diff
@@ -2331,22 +2330,6 @@ func skipIfNetworkTest(t testing.TB, actions []any) {
 	}
 }
 
-func ParseSDL(gqlSDL string) (map[string]client.CollectionDefinition, error) {
-	parser, err := graphql.NewParser()
-	if err != nil {
-		return nil, err
-	}
-	cols, err := parser.ParseSDL(gqlSDL)
-	if err != nil {
-		return nil, err
-	}
-	result := make(map[string]client.CollectionDefinition)
-	for _, col := range cols {
-		result[col.Description.Name.Value()] = col
-	}
-	return result, nil
-}
-
```

Comment:
question: What happened here? was this dead code?

Reply:
I changed the `Parser` interface and when I went to fix the function signature there I noticed it was dead code.